info.xml

```diff
@@ -14,14 +14,17 @@
     <url desc="Support">http://github.com/twomice/com.joineryhq.reltoken/issues</url>
     <url desc="Licensing">http://www.gnu.org/licenses/gpl-3.0.html</url>
   </urls>
-  <releaseDate>2021-12-27</releaseDate>
-  <version>0.3.1</version>
+  <releaseDate>2022-06-27</releaseDate>
+  <version>1.0</version>
   <develStage>beta</develStage>
   <compatibility>
-    <ver>5.0</ver>
+    <ver>5.43</ver>
```

Comment:
@MegaphoneJon Just curious: I'm clearly not up-to-date on all of this. Can you suggest docs where I can see the relevance of 5.43 here?

Reply:
Very cool, thanks @MegaphoneJon !